string name_motor
float32 pos
---
bool ret
